pydo/config/config.py

```diff
@@ -0,0 +1,28 @@
+import tempfile
+import os
+import json
+
+
+def load_config(path=None):
+    if path is None:
```

Comment:
What if path is not `None`? Looks like lines 9-13 should be unindented (and cf_path renamed to `path`)?